packages/scandipwa/src/component/ProductBundleOptions/ProductBundleOptions.style.scss

```diff
@@ -37,13 +37,6 @@
                 margin-block-start: 0;
             }
 
-            &_type_select + .Field-ErrorMessages {
-                position: absolute;
```

Comment:
The error label was positioned absolutely in order to display the error without shifting the contents down when the new element is rendered.
Did you check it?

Reply:
@aleksandrakorolova I can tell that page content is jumping up and down when an error message appears or disappears.
Is it expected behavior?

Reply:
Now the same behavior on the master. Decided to create separate task for this